src/components/Collection/SeriesInfo.tsx

```diff
@@ -128,12 +125,12 @@ const SeriesInfo = () => {
                 <div className="font-semibold ">Local</div>
                 <div className="flex flex-row gap-x-1">
                   <span>EP:</span>
-                  <span>{formatThousand(series.Sizes.Local.Episodes)}</span>
-                  {series.Sizes.Local.Specials !== 0 && (
+                  <span>{formatThousand((series?.Sizes.Local.Episodes) ?? 0)}</span>
```

Comment:
Looking a lot messier, from the docs it looks like we could use `initialData` to have some dummy object and not have to deal with undefined.

Reply:
I'll look into it 🤔

Reply:
It was more about typescript thinking series is null when it actually couldn't be. I fixed it without using initial data. But this initial data might be useful for other places 🙃

Reply:
It is because you're only check if the seriesId is nit falsely and not if the series is null when rendering the component (I think)